library/foundation/src/main/java/com/epmedu/animeal/foundation/spacer/Spacer.kt

```diff
@@ -0,0 +1,18 @@
+package com.epmedu.animeal.foundation.spacer
+
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.Dp
+
+@Composable
+fun HSpacer(height: Dp) {
+    Spacer(modifier = Modifier.height(height))
+}
+
+@Composable
+fun WSpacer(width: Dp) {
```

Comment:
it is worth avoiding abbreviations, perhaps it should have been called WidthSpacer or for ease of use and autocompletion SpacerWidth

Reply:
Done, renamed to WidthSpacer